mne/io/base.py

```diff
@@ -425,6 +425,30 @@ def _read_segment_file(self, data, idx, offset, fi, start, stop,
         raise NotImplementedError
 
     @verbose
+    @deprecated("This method has been renamed 'load_data' and will be removed "
+                "in v0.11.")
+    def preload_data(self, verbose=None):
```

Comment:
You'll also need to add this method name to the list of ignored things in `test_docstring_parameters`, it's why Travis is angry. Try running `nosetests mne/tests/test_docstring_parameters.py` and you'll see it.